### ISSUE TITLE
What can refer during the final?

### ISSUE THREAD

**Issue by Vincent6896** [nus-cs2030/2122-s1#375]
Hi. May I ask whether we can refer the codes submitted in codeCrunch, look the pyps and recitation answers during the final in the computer?

**Comment by swiftguy1** [comment]
yes to all i believe according to the luminus announcement

**Comment by Vincent6896** [comment]
> yes to all i believe, according to the luminus announcement

Thank you very much. One more question, can we refer to api like what we do in 2 two PEs?

**Comment by swiftguy1** [comment]
yes, use of vim, jshell and javac is also allowed according to this [https://github.com/nus-cs2030/2122-s1/issues/280#issuecomment-974617187](url)

**Comment by chenghong99** [comment]
> > yes to all i believe, according to the luminus announcement
> 
> Thank you very much. One more question, can we refer to api like what we do in 2 two PEs?

yes pretty much like PA

**Comment by rahulprasad01** [comment]
Thank you
